pymatgen/entries/corrections_calc.py

```diff
@@ -67,24 +67,22 @@ class CorrectionCalculator:
         "H",
     ]  # species that we're fitting corrections for
 
-    def __init__(self, exp_gz: str, comp_gz: str) -> None:
+    def __init__(self, max_error: float = 0.1, allow_unstable: bool = False, allow_polyanions: bool = False) -> None:
 
         """
         Initializes a CorrectionCalculator.
 
         Args:
-            exp_gz: name of gzip file that contains experimental data
-                    data in gzip file should be a list of dictionary objects with the following keys/values:
-                    {"formula": chemical formula, "exp energy": formation energy in eV/formula unit,
-                    "uncertainty": uncertainty in formation energy, "warnings": {"polyanion": str describing
-                    polyanions in compound or None, "large_uncertainty": str describing if compound has a large
-                    uncertainty value or None, "unstable": str describing if compound has high e above hull or None}}
-            comp_gz: name of gzip file that contains computed entries
-                    data in gzip file should be a dictionary of {chemical formula: ComputedEntry}
+            max_error: maximum tolerable relative uncertainty in experimental energy
+            allow_unstable: whether unstable entries are to be included in the fit
+            allow_polyanions: whether entries with problematic are to be included in the fit
```

Comment:
Let's avoid saying "problematic" and instead just say "entries containing polyanions (e.g., NO3, SO4, etc.)"

Reply:
Actually, examining the code below, can you just include the exact list of polyanions that the code searches for? "['SO4', 'CO3', 'NO3', 'OCl3', 'SiO4', 'SeO3', 'TiO3', 'TiO4']"

Reply:
and perhaps a short statement to the effect of "compounds with these polyanions have been observed to contain additional sources of error that may negatively influence the quality of the fitted corrections."